### PR TITLE
Normalize whitespace in evaluate CLI output test

### PR DESCRIPTION
**Pull request by adrianeboyd**
## Description

Depending on terminal settings, it looks like lines may be padded to the screen width so the comparison is too strict with only the command string replacement.

### Types of change

Tests.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy/tests/test_cli_app.py [hunk 1]
```diff
@@ -37,6 +38,8 @@ def test_benchmark_accuracy_alias():
     # Verify that the `evaluate` alias works correctly.
     result_benchmark = CliRunner().invoke(app, ["benchmark", "accuracy", "--help"])
     result_evaluate = CliRunner().invoke(app, ["evaluate", "--help"])
-    assert result_benchmark.stdout == result_evaluate.stdout.replace(
-        "spacy evaluate", "spacy benchmark accuracy"
+    assert re.sub(r"\s+", " ", result_benchmark.stdout) == re.sub(
+        r"\s+",
+        " ",
+        result_evaluate.stdout.replace("spacy evaluate", "spacy benchmark accuracy"),
```
Comment (svlandeg) [inline spacy/tests/test_cli_app.py:44]: Could we have this as a utility function in `tests\util.py` ?

Comment (adrianeboyd) [reply spacy/tests/test_cli_app.py:42]: I don't see a whole lot of advantage in moving that to a util method?

Comment (svlandeg) [reply spacy/tests/test_cli_app.py:42]: it's just that the `re.sub` command is copy/pasted in the current implementation, making it brittle to future changes IMO

Comment (adrianeboyd) [reply spacy/tests/test_cli_app.py:42]: I don't really care...

**Pull request merged**